### PR TITLE
Milestone1

### PR DESCRIPTION
**Pull request by pountzas**
We created the navbar and the footer with responsivness using bootstrap.

### CODE REVIEW

**Review by BrianSammit — CHANGES_REQUESTED** [review]
CHANGES REQUIRED :recycle: :red_circle: 
------------------
Hello @pountzas @arthurborgesdev :wave: You are doing an excellent job in this milestone. :+1: :muscle: 
There are just a few changes to make to complete this milestone :hammer_and_wrench: see them below :eyes: :point_down: 
------------

- [x] You have some minor errors in your commit messages. Remember that yous must write them in imperative. [Here](https://github.com/trein/dev-best-practices/wiki/Git-Commit-Best-Practices#formatting-rules) you can read about the formatting rules for commit messages. [Here](https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message) you can read how to changes commit messages. You don't need to changes these errors this time but keep these formatting rules in mind for the next ones. 
- [x] Check your `.gitignore`, you add a folder and still commit it, try to make an empty line at the end so that could solve the problem.
- [x] The Newsweek logo must be centered. Kindly fix it.
- [x] Login and Subscribe must be links `a` tag. 
- [x] On mobile the `Login` shouldn't appear.
- [x] It has a form with the search field. E.g `<input type="text" placeholder="Search">`
- [x] In mobile and table the menu hides and shows a hamburger button. 
- [x] In the footer section the Newsweek logo has a link `a` tag. 
- [x] For all  your empty href = " ". use a # to link to the current page. E.g  href = "#"
--------------------
Make the required changes :hammer_and_wrench: and send it for another review :eyes: :+1: 
HAPPY CODING!!

File: index.html
Comment (teekaytech) [inline index.html]: Using the `<span>` and `<p>` tags to list these items is not advisable. It is ideal to use the `<ul><li>` tag to list these items.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've made laudable progress in this milestone.
However, some changes are required before the milestone can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] The live demo link in your Readme is invalid. Kindly fix it.
- [x] On mobile and tablet screen sizes, the Newsweek image on the header should move to the left side. At the moment, the image only moves to the left on mobile view, but not on tablet view.
**Original Page**
orkgk a

**Your Page**
clone

- [x] Please check the comment below.

**[ OPTIONAL CHANGES ]**

- [x] The goal of this project is to master bootstrap by producing a similar webpage to the given Newsweek website. Therefore, you should pay attention to some details, like color, spacing, margin, and padding. For instance, the red color in your header is not the same as what we have on the original website.


* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations team :tada: 

**This project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:**


Again, congratulations, you can merge to the master branch.
Do not forget to fill the project completion form.

Good luck with your next milestone :blush:

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by elmejdki — COMMENTED** [review]
# status: lnvalid :red_circle: 

Really sorry guys I had to mark your project as invalid :disappointed: because your branch `milestone1` was removed so I have nothing to review, please put your changes in another branch it's better if you named it `milestone1` and then ask for another code review.

## Happy Coding :tophat:
**_Zakariae El Mejdki_**

**Pull request merged**
